packages/storybook/src/semantic-ui/FileInputButton.stories.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import React, { useCallback } from 'react';
+import { withKnobs } from '@storybook/addon-knobs';
+import FileInputButton from '../../../semantic-ui/src/components/FileInputButton';
+
+export default {
+  title: 'Components/Semantic UI/FileInputButton',
+  component: FileInputButton,
+  decorators: [withKnobs]
+};
+export const Default = () => {
+  const onSelection = useCallback((item) => {
+    console.log(item);  
+  });
+
+  return (
+    <FileInputButton
+      content='Select a file to upload'
+      icon='times'
+      multiple={false}
+      onSelection={onSelection}
```

Comment:
Here we can use the `action` function to log events to the "Actions" pane instead of the console. There's a good example in the [FileUpload.stories.js](https://github.com/performant-software/react-components/blob/be2ad95821d153330556ed20fb9692388b8d73b1/packages/storybook/src/semantic-ui/FileUpload.stories.js#L1).

Reply:
Done.